### ISSUE TITLE
Listmasternodes command not working as it should and not showing consistent results

### ISSUE THREAD

**Issue by oriz456** [Divicoin/Divi#81]
I set up my masternode, per instructions here: https://wiki.diviproject.org/#masternode-setup. Started the daemon by adding the addnode command, both in local as well as the remote host. Waited till getstaking status returned mnsync = true(in fact, in local everything returns true, while on remote all but stakingstatus and mintablecoins come back true), ran startmasternode command. Masternode started, but activetime never moved from 0, even though rewards were coming in. As can be seen in the screenshot(mine is the last masternode on the list here, address starting with DN), the activetime is 0 and stays there no matter how much time passes, and on the right hand side can be seen the remote server which returns empty brackets. 

![ea0f0ae48f0188265be1b427b605e45c5828d29d](https://user-images.githubusercontent.com/32154971/44301552-e7352e00-a2e6-11e8-8e06-ef685d33888d.png)

As an additional issue that I never had before while the other things were happening, I just got back after I was away for a couple of hours and even listmasternodes locally would return empty brackets, but in this case listunspent kept working and balance kept going up. I started the masternode again, and now listmasternodes works but only lists mine. See screenshot below

![image](https://user-images.githubusercontent.com/32154971/44301589-b275a680-a2e7-11e8-8ee5-a1c03eacc13a.png)

**Comment by oriz456** [comment]
I should note this happens with Windows.

**Comment by oriz456** [comment]
Experimenting also with a few things, like the long divi.conf version in local vs remote, I had one masternode that has both long(with the external ip and key), one has the long one in remote and short in local(the way it's originally supposed to be per the guide) and one the other way around. surprisingly, the most stable ones, that have taken much longer to expire  but kept receiving rewards(see comment in issue #82) are actually the ones with the long version in LOCAL. The one that was configured correctly kept expiring more frequently and needing startmasternode to be called much more often. 



Activetime coming back 0, seems to have gotten fixed when I changed the divi.conf configuration to actually have the "longer version" in local(see comment in issue #81). Interestingly, prior to that experiemnt while activetime was always 0, lastpaid was going up. However, after the change activetime would go up, but lastpaid stayed at 0. I am not sure it is unique to this situation though, as other masternodes now appear to be showing lastpaid as 0 that aren't mine and that didn't show it before. Other testers have noticed this as well . lastpaid in most masternodes usually comes back as 0, even the ones that had activetime fixed per comment in issue #81. This is despite the fact that they are, indeed getting paid. 
Calling listmasternodes from the remote server returned empty brackets in all but the diamond server. Initially I had believed it to be due to the configuration changes I had made(see comment in issue #81), but even once I changed it all back, it will still list them, although listunspent doesn't work in ANY remote server. 

Additionally, the results are inconsistent, and will at times show all masternodes, at times just some of them, etc.
